src/components/Footer/index.js

```diff
@@ -37,10 +36,8 @@ const Footer = () => {
   const mapLayers = new MapLayers()
   const { isAuthenticated } = useAuth0()
 
-  const downloadReport = () => {
-    const date = dateFormat(dateContext)
-
-    mapLayers.downloadReport(countryContext, date)
+  const downloadMapLayerReport = () => {
```

Comment:
This name isn't really accurate -- I'd keep it just `downloadReport`

Reply:
no problems